dev-docs/bidders/sparteo.md

```diff
@@ -27,12 +27,14 @@ Sparteo Header Bidding adapter requires setup and approval. Please reach out to
 ### Bid params
 
 {: .table .table-bordered .table-striped }
-| Name          | Scope    | Description                                                                             | Example    | Type      |
-|---------------|----------|-----------------------------------------------------------------------------------------|------------|-----------|
-| `publisherId` | required | Your publisher ID. This information will be given to you by the Sparteo team.           | `1234`     | `integer` |
-| `networkId`   | required | Your network ID. This information will be given to you by the Sparteo team.             | `1234`     | `integer` |
-
-Only the `publisherId` OR `networkId` is required.
+| Name          | Scope    | Description                                                                                                                         | Example                                  | Type      |
+|---------------|----------|-------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------|-----------|
+| `networkId`   | required | Your network ID. This information will be given to you by the Sparteo team.                                                         | `'9061ab7e-f01e-43a5-b801-c1a56156f105'` | `String`  |
```

Comment:
This is out of sync with your adapter code, which suggests that either networkId or publisherId or required:
```
    if (!bid.params.networkId && !bid.params.publisherId) {
      logError('The networkId or publisherId is required');
      return false;
    }
```

Reply:
Oh thank you! Good catch, I missed it. I'll open another Pull Request against Prebid JS and adapt this one with the link when done.

Reply:
@bretg I created a PR in Prebid.js to fix it and updated this PR description: https://github.com/prebid/Prebid.js/pull/12321

Thank you for the review!